recipe/meta.yaml

```diff
@@ -17,22 +17,22 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy <1.14  # [py27]
-    - numpy >1.7.1  # [not py27]
-    - astropy >=2.0
+    - numpy >=1.11|!=1.14.0  # [py27]
+    - numpy >=1.11  # [not py27]
```

Comment:
You should build with the oldest version of numpy that you want to support and keep the run requirements as it is here.

```
build:
  - numpy 1.11.*
run:
  - numpy >=1.11|!=1.14.0  # [py27]
  - numpy >=1.11  # [not py27]
```

Reply:
Yes that is a good point. I will update it for the next go around. Thanks!